src/main/webapp/app/exercises/text/participate/text-editor.component.html

```diff
@@ -32,10 +32,10 @@
                 }
                 <div class="grid-area-main">
                     <div>
-                        <span class="badge bg-primary mb-2" id="word-count">
+                        <span class="badge bg-primary mb-2" id="word-count" [ngClass]="{ 'no-submission': submission && !submission.submitted }">
```

Comment:
Can you use `[hidden]` here instead of a class with `display: none;`? I think that makes it more clear what you want to achieve.

Reply:
You are right, I integrated your suggestion now. I think that class will be necessary in `modeling-submission.component.html`, since we need `!important` there to overwrite it.

Reply:
Can you try that out? I believe that should still work since hidden elements will be completely removed, but I'm not sure.

Reply:
I tried it now and didn't work 😄 
<img width="1253" alt="Screenshot 2024-07-17 at 15 42 24" src="https://github.com/user-attachments/assets/e31d303b-561d-42c1-9f6b-fcd4e424c830">